CONTRIBUTING.md

```diff
@@ -59,25 +68,55 @@ Code review is a crucial aspect of contributing to a project, and all contributo
 
 ## Design document
 
-Write down your implementation plan and discuss it with other developers in the community before you start coding officially.
-If it is just a small changes, describe the implementation steps clearly in the Issue.
-If it is a relatively large work, it is recommended to write a design document for this feature.
-Here is a [design document template](https://docs.google.com/document/d/1LeTyrlzQJfSs2DkRBsucK_vV5gtHRYLb1KSrpu0hp3g/edit?usp=sharing) for reference.
+Write down your implementation plan and discuss it with other developers in the community before you
+start coding officially. If it is just a small change, describe the implementation steps clearly in
+the Issue. If it is a relatively large work, it is recommended to write a design document for this
+feature. Here is
+a [design document template](https://docs.google.com/document/d/1LeTyrlzQJfSs2DkRBsucK_vV5gtHRYLb1KSrpu0hp3g/edit?usp=sharing)
```

Comment:
The design document template should be rename from `Arctic` to `Amoro`.

Reply:
Thanks for the reminder, I have changed the doc name and translated the doc to the English version.